first_hour_on_vwb/working_with_data_collections.ipynb

```diff
@@ -103,16 +104,17 @@
     "tags": []
    },
    "source": [
-    "## Create a data collection\n",
+    "## Create a new data collection\n",
+    "<a href=\"create-new-dc\"></a>\n",
     "\n",
-    "The process of creating a data collection requires you to specify the following:\n",
+    "Before creating your data collection, consider the following:\n",
     "1. What data do you want to share? What type of resources--Cloud Storage buckets or objects, BigQuery tables, GitHUb repositories--will be made available via this data collection?\n",
     "1. With whom do you wish to share this data? Will you be sharing the data collection with all members of an existing Verily Workbench group (e.g. for your organization or team), or will you need to create a new Verily Workbench group in order to restrict access to the data collection?\n",
-    "1. Will you update the data collection by releasing future versions?\n",
+    "1. Will you update the data collection by releasing future versions? What versioning scheme is most appropriate?\n",
     "\n",
     "<div class=\"alert alert-block alert-success\">\n",
     "<b>Note:</b> \n",
-    "    If you'd like to restrict access to your data collection to members of a specific group, you'll need to provide the <a href=\"https://et-docs-tests.googleplex.com/docs/reference/glossary/#policy\">group policy constraint</a> at the time of workspace creation. See <a href=\"../creating_a_group.ipynb\">../creating_a_group.ipynb</a> for details on how to create a Verily Workbench group that can be used for group policy constraints on workspaces and data collections. </div>"
+    "    If you'd like to restrict access to your data collection to members of a specific group, contact <a href=\"mailto:workbench-support@verily.comworkbench-support@verily.com\">Verily Workbench Support</a> <b>before</b> creating your workspace so that a <a href=\"https://et-docs-tests.googleplex.com/docs/reference/glossary/#policy\">group policy constraint</a> can be applied at the time of workspace creation. See <a href=\"../creating_a_group.ipynb\">../creating_a_group.ipynb</a> for details on how to create a Verily Workbench group that can be used for group policy constraints on workspaces and data collections. </div>"
```

Comment:
Is the formatting of the support email address correct? Looks like its pasted twice

Reply:
done!